src/background/utils.ts

```diff
@@ -234,3 +234,53 @@ export async function fetchCurrentBlockNumber(): Promise<number> {
     return -1;
   }
 }
+
+export type GasPrices = {
+  gasPriceWei: BigInt;
+  gasPriceWei_data: BigInt;
+  gasPriceFri: BigInt;
+  gasPriceFri_data: BigInt;
+};
+
+export async function fetchCurrentGasPrices(): Promise<GasPrices | null> {
+  try {
+    const provider = await getProvider();
+    const latestBlock = await provider.getBlockLatestAccepted();
+    const blockDetails = (await provider.getBlock(latestBlock.block_number)) as any;
+
+    console.log('Latest block info:');
```

Comment:
leftover?

Reply:
y, from testing/demoing the gas price block issue